proto/dd_trace.proto

```diff
@@ -16,6 +17,36 @@ message APITrace {
 	int64 endTime = 7;
 }
 
+// AgentPayload is the top level container for the newest Datadog trace format
+message AgentPayload {
+	string hostName = 1;
+	string env = 2;
+	repeated TracerPayload tracerPayloads = 5;
+	map<string, string> tags = 6;
+	string agentVersion = 7;
+	double targetTPS = 8;
+	double errorTPS = 9;
```

Comment:
This is the exact same message as the one use by the `trace-agent` (https://github.com/DataDog/datadog-agent/blob/main/pkg/trace/pb/agent_payload.proto).

Reply:
Yeah, fair enough. I reached out (and CC'd you) on a thread with the APM intake team because I'm really curious about this... 🤔  I'll consider this comment non-blocking, though, for the purpose of the review.